src/routes/game/components/dialogs/components/OpponentRequestedStalemateDialog.vue

```diff
@@ -88,7 +88,7 @@ export default {
     async acceptStalemate() {
       this.loadingAccept = true;
       try {
-        await this.gameStore.requestStalemate();
+        await this.gameStore.acceptStalemate();
```

Comment:
Created a separate store action for accepting stalemate vs requesting one initially to make legacy API more consistent w/ GameState API, which treats these as two separate actions